app.js

```diff
@@ -2,6 +2,25 @@ const express = require("express");
 const app = express();
 app.use(express.json());
 
+const { getTopics } = require("./_controllers/topics.controllers");
+
+app.get("/api/topics", getTopics);
+
+app.use((err, req, res, next) => {
+  if (err.status) {
+    res.status(err.status).send({ msg: err.msg });
+  } else next(err);
+});
+app.use((err, req, res, next) => {
+  if (err.code === "22P02") {
```

Comment:
Great that your handling errors, but you should only have the code you need for the specific ticket/functionality.